scout/server/blueprints/variant/templates/variant/variant_details.html

```diff
@@ -138,17 +138,6 @@
         </tbody>
       </table>
     </div>
-    {% if variant.track == "rd" %}
```

Comment:
This was not used and not working anyway

Reply:
As for the placement, what made it weigh over for the Summary card? I would say thematically it fits the "current" Frequencies better. Layout wise, the ClinVar table in Summary can grow, but on most variants it would be short so that Summary and the up to three cards on freq end up roughly the same size. Touch and go there. Without more arguments, I weigh towards keeping these links in Frequencies.